server/handlers.go

```diff
@@ -112,7 +115,9 @@ func (a *app) auth(w http.ResponseWriter, r *http.Request) {
 }
 
 func (a *app) index(w http.ResponseWriter, r *http.Request) {
+	log.Printf("Entering index handler.")
 	tok := a.getAuthToken(r)
+	log.Printf("Token found: %v\n", tok)
```

Comment:
I don't think the token should be logged.

Reply:
Oops! No, sorry, that was a debug thing.  gone.